lib/browser/rpc-server.js

```diff
@@ -465,6 +465,10 @@ ipcMain.on('ELECTRON_BROWSER_WINDOW_CONFIRM', function (event, message, title) {
 
 // Implements window.close()
 ipcMain.on('ELECTRON_BROWSER_WINDOW_CLOSE', function (event) {
-  event.sender.getOwnerBrowserWindow().close()
```

Comment:
`try {} catch (err) {}` is quite costly if I recall correctly.

Can we just check if the window is `null` or not?  I.e.

```js
const window = event.sender.getOwnerBrowserWindow();
if (window) {
  window.close()
}
event.returnValue = null
```